code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -865,4 +865,12 @@ var/list/ai_list = list()
 /mob/living/silicon/ai/replace_identification_name(oldname,newname)
 	if(aiPDA)
 		aiPDA.owner = newname
-		aiPDA.name = newname + " (" + aiPDA.ownjob + ")"
\ No newline at end of file
+		aiPDA.name = newname + " (" + aiPDA.ownjob + ")"
+
+
+/mob/living/silicon/ai/proc/add_malf_picker()
+	src << "In the top right corner of the screen you will find the Malfunctions tab, where you can purchase various abilities, from upgraded surveillance to station ending doomsday devices."
+	src << "You are also capable of hacking APCs, which grants you more points to spend on your Malfunction powers. The drawback is that a hacked APC will give you away if spotted by the crew. Hacking an APC takes 60 seconds."

```

Comment:
I just moved this stuff out of the traitor AI setup into a proc, it's been like this for years. 

What class should it use?